src/models/Order.ts

```diff
@@ -49,6 +53,16 @@ Order.init(
       type: DataTypes.STRING,
       allowNull: false,
     },
+    createdAt: {  // Explicitly define createdAt field
+      type: DataTypes.DATE,
+      allowNull: false,
+      defaultValue: DataTypes.NOW, // Set to current timestamp by default
+    },
+    updatedAt: {  // Explicitly define updatedAt field
```

Comment:
This is not important, no need to change it, but keep it in mind as a note. There's no need to add all this code; it's enough to just include:  
```javascript
{
  sequelize,
  tableName: "products",
  modelName: "Product",
}
```